Formula/shapeit4.rb

```diff
@@ -0,0 +1,24 @@
+class Shapeit4 < Formula
+  desc "Segmented HAPlotype Estimation and Imputation Tool"
+  homepage "https://odelaneau.github.io/shapeit4/"
+  url "https://github.com/odelaneau/shapeit4/archive/refs/tags/v4.2.2.tar.gz"
+  sha256 "9f109e307b5cc22ab68e7bf77de2429a9bbb2212d66303386e6a3dd81a5bc556"
+  license "MIT"
+
+  depends_on "boost"
+  depends_on "htslib"
+
+  def install
+    system "sed -i'' -e 's/setiosflags/std::setiosflags/g' src/utils/string_utils.h"
+    system "sed -i'' -e 's/setiosflags/std::setiosflags/g' tools/bingraphsample/src/utils/string_utils.h"
```

Comment:
try to use `inreplace` to do the code changes.

Also, is there any upstream issue for such patching work?